marketing/adhoc-email-performance/sendGridEmailPerformance.py

```diff
@@ -0,0 +1,54 @@
+import os
+import json
+import time
+import python_http_client
+import pandas as pd
+from pandas.io.json import json_normalize
+
+# initialize client
+with open(os.path.join(os.path.realpath('/Users/fronk/.creds'), 'sendgrid_creds.json'), 'r') as f:
+    config = json.load(f)
+api_key = config['key']
+host = 'https://api.sendgrid.com'
+request_headers = {'Authorization': 'Bearer {0}'.format(api_key)}
+version = 3
+client = python_http_client.Client(
+	host=host,
```

Comment:
tabs?! 😛

Reply:
😬 :slowly backs away into the bushes: - fixed :)